src/main/java/be/minelabs/Minelabs.java

```diff
@@ -53,6 +54,7 @@ public void onInitialize() {
         CraftingRecipes.onInitialize();
         Villagers.onInitialize();
 
+        MinelabsGameRules.onInitialize();
```

Comment:
Our naming convention up until now has been to use the same names as vanilla MC. Please rename to `GameRules`

Reply:
Didn't want to have to write it like this every time. 
<img width="887" alt="image" src="https://github.com/ScicraftLearn/Minelabs/assets/7142721/d3120bad-b224-4b07-9298-b9d3f6629e01">

Reply:
Fair enough